openedx/adg/lms/applications/models.py

```diff
@@ -138,6 +146,8 @@ class UserApplication(TimeStampedModel):
         help_text=_('Accepted extensions: .pdf, .doc, .jpg, .png'),
     )
     cover_letter = models.TextField(blank=True, verbose_name=_('Cover Letter'), )
+    is_work_experience_not_applicable = models.BooleanField(verbose_name=_('Work Experience Not Applicable'),
+                                                            default=False)
```

Comment:
Can we break after the opening parentheses? Suggested by [edX](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines).